articles/portal/ptl-how-restrict-access-vcloud-api.md

```diff
@@ -1,36 +1,37 @@
 ---
-title: How to restrict access to the vCloud API
-description: Shows how to create a whitelist of IP addresses to restrict access to the vCloud API
+title: How to restrict access to VMware Cloud Director/vCloud Director
+description: Shows how to create an allowlist of IP addresses to restrict access to the vCloud API and VMware Cloud Director/vCloud Director
 services: portal
-author: Sue Highmoor
-reviewer:
-lastreviewed: 15/11/2019
+author: shighmoor
+reviewer: shighmoor
+lastreviewed: 01/12/2020
 
 toc_rootlink: How To
 toc_sub1: 
 toc_sub2:
 toc_sub3:
 toc_sub4:
-toc_title: Restrict access to the vCloud API
+toc_title: Restrict access to vCloud Director
 toc_fullpath: How To/ptl-how-restrict-access-vcloud-api.md
 toc_mdlink: ptl-how-restrict-access-vcloud-api.md
 ---
 
-# How to restrict access to the vCloud API
+# How to restrict access to VMware Cloud Director/vCloud Director
 
 ## Overview
 
-You can restrict access to the vCloud API by creating a whitelist of acceptable IP addresses. This helps to prevent access to the API from unauthorised sources as users will only be able to access the API if their request originates from an IP address on the whitelist.
+You can restrict access to VMware Cloud Director/vCloud Director by creating an allowlist of acceptable IP addresses. This helps to prevent access from unauthorised sources as users will only be able to access VMware Cloud Director/vCloud Director if their request originates from an IP address on the allowlist.
+
+> [!NOTE]
+> Enabling this option restricts access to both the vCloud API and the VMware Cloud Director/vCloud Director UI.
 
 > [!NOTE]
 > When specifying IP addresses, you must use individual IP addresses. You cannot restrict access to groups of IP addresses, for example by range or subnet.
 
 > [!IMPORTANT]
-> Be careful when entering IP addresses. Entering the wrong address may make the vCloud API inaccessible and you will be unable to change the settings. If you lose access to the API, contact UKCloud Support.
-
-## Restricting access to the vCloud API
+> Be careful when entering IP addresses. Entering the wrong address may make the API and UI inaccessible and you will be unable to change the settings. If you lose access, contact UKCloud Support.
```

Comment:
I think this warning should go against restricting access to the Portal instead of restricting access to vCloud.

If you get your vCloud IP wrong you can easily enough update it in the Portal.

If you get your Portal IP wrong, then you can't easily get to the Portal to update it.

Reply:
We do have the following note in the Portal article:
Be careful when entering IP addresses. Entering the wrong address may make the Portal inaccessible and you will be unable to change the settings. If you lose access to the Portal, contact UKCloud Support.
So I think we're covered there.
I do think it's still worth noting in the vCloud article - maybe I'll downgrade it to a regular note rather than Important note

Reply:
I've also taken out the bit about having to contact us to fix it as the customer will still be able to access the Portal to make the changes themselves

Reply:
My mistake- so we do.

I like your recent change